app/web/platform/nativeLink.ts

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+
+export function getReactNativeWebView(): typeof window.ReactNativeWebView {
+  if (
+    typeof window !== "undefined" &&
+    window.ReactNativeWebView !== undefined
```

Comment:
I think this can just be `typeof window && window.ReactNativeWebView`. You can see it here or test yourself in the dev console.

<img width="411" alt="Screenshot 2024-08-29 at 8 25 33 PM" src="https://github.com/user-attachments/assets/8ea1ccc7-e8ae-4da2-8b64-fe143f1af11b">

Reply:
fixed